src/components/searchbutton/SearchButton.scss

```diff
@@ -28,5 +30,18 @@
     display: -webkit-box;
     -webkit-line-clamp: 2;
     -webkit-box-orient: vertical;
+    line-height: 20px;
```

Comment:
better to use decimal than pixels. [info](https://dev.to/lampewebdev/css-line-height-jjp )